src/app/search/search-filters.js

```diff
@@ -448,7 +448,9 @@ export function searchFilters(searchConfig, storeService, toaster, $i18next) {
               dupes.sort(dupeComparator);
               const bestDupe = dupes[0];
               for (const dupe of dupes) {
-                _lowerDupes[dupe.id] = dupe !== bestDupe;
+                if (dupe.weaponOrArmor) {
```

Comment:
`if (dupe.equipment) {`
then the rest are unneeded changes

Reply:
transferable? can't find this.
even then, that would still affect ghosts, doesn't?

Reply:
Irt #2684 weapons & gear ghosts, ships, and sparrows all fall under this category...

Reply:
@duckmanBR there is no need to reinvent the wheel
`if item.bucket && (item.bucket.sort === 'Weapons' || item.bucket.sort === 'Armor')`

Reply:
But then I could just use the code of the isWeaponOrArmor function (that wasn't created by me btw)  
`if (dupe.primStat && [1480404414, 3897883278].includes(dupe.primStat.statHash)) {`
But if I include a `&& !dupe.notransfer` on your code they would have the same result (that is needed because you could have the same quest on multiple characters).

I only added a property (even though I knew that would increase the footprint) because I thought that having the same logic spread around was a bad thing.

I'll revert those later.